src/context/mod.rs

```diff
@@ -480,6 +495,27 @@ impl Context {
         unsafe { raw::RedisModule_ReplyWithError.unwrap()(self.ctx, msg.as_ptr()).into() }
     }
 
+    #[cfg(feature = "min-redis-compatibility-version-8-0")]
```

Comment:
but isn't `RedisModule_AddACLCategory` present in Valkey 7.2?

Reply:
https://github.com/valkey-io/valkey/releases/tag/8.0.0-rc1
From my understanding it was released with the above version.

Reply:
This is the right file to review, https://github.com/valkey-io/valkey/blob/unstable/src/valkeymodule.h. The redismodule.h is just a shim for compatibility.

Reply:
thank you @madolson.  So it's only 8.0 that has actual capability to add ACL categories.